src/apps/competitions/tests/test_submissions.py

```diff
@@ -213,3 +216,29 @@ def test_making_submission_to_phase_with_one_task_does_not_create_parents_or_chi
         assert resp.call_count == 1
         sub = Submission.objects.get(id=self.sub.id)
         assert not sub.has_children
+
+    def test_adding_task_to_phase_runs_submissions_on_new_task(self):
+
```

Comment:
yoctoscopic nitpick: delete empty line between function prototype line and implementation lines.